Leanplum/Messages/LPMessageDisplayManager.m

```diff
@@ -0,0 +1,33 @@
+//
+//  LPMessageDisplayManager.m
+//  Leanplum
+//
+//  Created by Mayank Sanganeria on 11/13/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import "LPMessageDisplayManager.h"
+#import "LPAlertMessage.h"
+
+@implementation LPMessageDisplayManager
+
+-(UIAlertController *)create:(LPAlertMessage *)message {
+    UIAlertController * alert=   [UIAlertController
+                                  alertControllerWithTitle:message.title
+                                  message:message.message
+                                  preferredStyle:UIAlertControllerStyleAlert];
+
+    UIAlertAction *dismiss = [UIAlertAction
+                             actionWithTitle:message.dismissText
+                             style:UIAlertActionStyleDefault
+                             handler:^(UIAlertAction * action) {
+        [alert dismissViewControllerAnimated:YES completion:nil];
+    }];
+    [alert addAction:dismiss];
+    return alert;
+}
+
+-(void)display:(LPAlertMessage *)message {
```

Comment:
This is a great start. I think we need to also do this end to end with the lower level stitching of the code.